### PR TITLE
fix: add number of lines stats per commit, add binary filter, other minor fixes

### PR DESCRIPTION
**Pull request by anatolykislov**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -63,6 +63,7 @@ dependencies {
     compile 'com.github.kittinunf.fuel:fuel-rxjava:1.9.0'
     compile group: 'org.eclipse.jgit', name: 'org.eclipse.jgit',
             version: '4.8.0.201706111038-r'
+    compile "org.slf4j:slf4j-nop:1.7.2"
```
Comment (asurkov) [inline build.gradle]: not sure how this exactly works, I trust you though on this one.

File: src/main/proto/sourcerer.proto [hunk 2]
```diff
@@ -23,7 +23,7 @@ message Commit {
     string author_name = 4;
     string author_email = 5;
 
-    // Timestamp of a commit creation.
+    // Timestamp of a commit creation in seconds UTC.
```
[the remark this answers is not in the document]
Comment (anatolykislov) [reply src/main/proto/sourcerer.proto]: I mean UTC+00:00.

File: build.gradle (hunk 1 above)
Comment (anatolykislov) [reply build.gradle]: JGit use SLF4J API, nop is an empty implementation of this api (stands for no operation).

File: src/main/kotlin/app/extractors/ObjectiveCExtractor.kt [hunk 3]
```diff
@@ -10,7 +10,7 @@ import app.model.DiffFile
 class ObjectiveCExtractor : ExtractorInterface {
     companion object {
         val LANGUAGE_NAME = "objectivec"
-        val FILE_EXTS = listOf("h", "m", "mm")
+        val FILE_EXTS = listOf("m", "mm")
```
[the remark this answers is not in the document]
Comment (anatolykislov) [reply src/main/kotlin/app/extractors/ObjectiveCExtractor.kt]: Yes, but for now we can't distinguish h file from C/C++ or ObjectiveC. This extension will be introduced with ANTLR parsing of files.

**Pull request merged**